### PR TITLE
GA4: Enable Google Signals

### PR DESCRIPTION
**Pull request by ishachopra0401**
## Description of the change

Enable Google Signals

## Type of change
- [ ] Bug fix (non-breaking change that fixes an issue)
- [ ] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Related issues

> Fix [#1]() 

### Development

- [ ] Lint rules pass locally
- [ ] The code changed/added as part of this pull request has been covered with tests
- [ ] All tests related to the changed code pass in development

### Code review 

- [ ]  This pull request has a descriptive title and information useful to a reviewer. There may be a screenshot or screencast attached
- [ ] "Ready for review" label attached to the PR and reviewers mentioned in a comment
- [ ] Changes have been reviewed by at least one other engineer
- [ ] Issue from task tracker has a link to this pull request

### CODE REVIEW

**Review by rohithkaza — APPROVED** [review]
LGTM

File: docs/destinations/analytics/google-analytics-4/google-analytics-4-device-mode.mdx [hunk 1]
```diff
@@ -111,6 +111,32 @@ Below are some examples of the track event names that are passed to Google Analy
 For each product in the order, there must be an <code class="inline-code">id</code> and <code class="inline-code">name</code> associated. More info on Google Analytics 4 e-commerce event and corresponding properties can be found <a href="https://developers.google.com/gtagjs/reference/ga4-events#view_item_list">here</a>.
 </div>
 
+## FAQ
+
+### How can I enable Google Signals in GA4?
+
+[Google Signals](https://support.google.com/analytics/answer/9445345?hl=en#zippy=%2Cin-this-article) helps you understand the interaction of users with your website across multiple devices and sessions. To enable Google Signals and capture user data, the following conditions must be true:
```
Comment (utsabc) [inline docs/destinations/analytics/google-analytics-4/google-analytics-4-device-mode.mdx:118]: Can we mention that this is device mode only feature?

File: docs/destinations/analytics/google-analytics-4/google-analytics-4-device-mode.mdx [hunk 2]
```diff
@@ -111,6 +111,37 @@ Below are some examples of the track event names that are passed to Google Analy
...
```
Comment (ameypv-rudder) [inline docs/destinations/analytics/google-analytics-4/google-analytics-4-device-mode.mdx:118]:
```suggestion
[Google Signals](https://support.google.com/analytics/answer/9445345?hl=en#zippy=%2Cin-this-article) helps you understand the interaction of users with your website across multiple devices and sessions. To enable Google Signals and capture user data, the following conditions must be met:
```

File: docs/destinations/analytics/google-analytics-4/google-analytics-4-device-mode.mdx [hunk 3]
```diff
@@ -111,6 +111,37 @@ Below are some examples of the track event names that are passed to Google Analy
...
+- A user must be signed in to Google Account on a website or an app.
```
Comment (ameypv-rudder) [inline docs/destinations/analytics/google-analytics-4/google-analytics-4-device-mode.mdx:119]:
```suggestion
- A user must be signed in to their Google Account on the website or app.
```

File: docs/destinations/analytics/google-analytics-4/google-analytics-4-device-mode.mdx [hunk 4]
```diff
@@ -111,6 +111,37 @@ Below are some examples of the track event names that are passed to Google Analy
...
+- Ad Personalization must be turned on for their account.
```
Comment (ameypv-rudder) [inline docs/destinations/analytics/google-analytics-4/google-analytics-4-device-mode.mdx:120]:
```suggestion
- Ad Personalization must be enabled for their account.
```

File: docs/destinations/analytics/google-analytics-4/google-analytics-4-device-mode.mdx [hunk 5]
```diff
@@ -111,6 +111,37 @@ Below are some examples of the track event names that are passed to Google Analy
...
+
+<div class="infoBlock">
+
+This feature is available in the web <Link to="/rudderstack-cloud/rudderstack-connection-modes/#device-mode">device mode</Link> only.
```
Comment (ameypv-rudder) [inline docs/destinations/analytics/google-analytics-4/google-analytics-4-device-mode.mdx:124]:
```suggestion
The Google Signals feature is available in the web <Link to="/rudderstack-cloud/rudderstack-connection-modes/#device-mode">device mode</Link> only.
```

File: docs/destinations/analytics/google-analytics-4/google-analytics-4-device-mode.mdx [hunk 6]
```diff
@@ -111,6 +111,37 @@ Below are some examples of the track event names that are passed to Google Analy
...
+</div>
+
+To enable Google Signals in GA4 and reflect the collected user data in your GA4 reports, follow the steps below:
```
Comment (ameypv-rudder) [inline docs/destinations/analytics/google-analytics-4/google-analytics-4-device-mode.mdx:127]:
```suggestion
To enable Google Signals in GA4 and reflect the collected user data in your reports, follow the steps below:
```

File: docs/destinations/analytics/google-analytics-4/google-analytics-4-device-mode.mdx [hunk 7]
```diff
@@ -111,6 +111,37 @@ Below are some examples of the track event names that are passed to Google Analy
...
+
+1. Log into your [Google Analytics dashboard](https://analytics.google.com/). 
+2. Open the Admin panel by clicking on the gear icon at the bottom left.
+<img src="../../../assets/event-stream-destinations/Google-signals-1.png" alt="Google Signals" />
+
+3. Select the property for which you want to enable Google Signals. Then, go to **Data Settings** > **Data Collection**.
+4. Click **Get Started** in the **Google signals data collection** section, as shown:
+<img src="../../../assets/event-stream-destinations/Google-signals-2.png" alt="Google Signals" />
+
+6. Click on **Continue**.
```
Comment (ameypv-rudder) [inline docs/destinations/analytics/google-analytics-4/google-analytics-4-device-mode.mdx:137]:
```suggestion
6. Click **Continue**.
```

File: docs/destinations/analytics/google-analytics-4/google-analytics-4-device-mode.mdx [hunk 8]
```diff
@@ -111,6 +111,37 @@ Below are some examples of the track event names that are passed to Google Analy
...
+<img src="../../../assets/event-stream-destinations/Google-signals-3.png" alt="Google Signals" />
+
+7. Read the additional information and click on **Activate**.
```
Comment (ameypv-rudder) [inline docs/destinations/analytics/google-analytics-4/google-analytics-4-device-mode.mdx:140]: we should be consistent with "Click" or "Click on" - something to consider for the style guide.

File: docs/destinations/analytics/google-analytics-4/google-analytics-4-device-mode.mdx [hunk 9]
```diff
@@ -111,6 +111,37 @@ Below are some examples of the track event names that are passed to Google Analy
...
+<img src="../../../assets/event-stream-destinations/Google-signals-4.png" alt="Google Signals" />
+
+Google Signals is now enabled. You will be able to see the **Age**, **Gender**, and other relevant data in GA4 Demographics reports after 24 hours.
```
Comment (ameypv-rudder) [inline docs/destinations/analytics/google-analytics-4/google-analytics-4-device-mode.mdx:143]: GA4 Demographics reports -- is this the name of the report? else we can write 'demographics', i.e. in lower case.

**Review by ameypv-rudder — APPROVED** [review]
Good to go once we incorporate the minor changes.

File: docs/destinations/analytics/google-analytics-4/google-analytics-4-device-mode.mdx (hunk 8 above)
Comment (ishachopra0401) [reply docs/destinations/analytics/google-analytics-4/google-analytics-4-device-mode.mdx:140]: agree.. I personally prefer "Click" (minimal choice) but used "Click on" to be consistent.
Can take a call surely!

File: docs/destinations/analytics/google-analytics-4/google-analytics-4-device-mode.mdx (hunk 9 above)
Comment (ishachopra0401) [reply docs/destinations/analytics/google-analytics-4/google-analytics-4-device-mode.mdx:143]: Checked here: https://support.google.com/analytics/answer/2819948?hl=en#zippy=%2Cin-this-article
Seems like "Demographics" should be fine

**Pull request merged**